package/firmware/ipq-wifi/Makefile

```diff
@@ -29,15 +29,14 @@ ALLWIFIBOARDS:= \
 	devolo_magic-2-wifi-next \
 	edgecore_ecw5410 \
 	edgecore_oap100 \
-	extreme-networks_ws-ap3915i \
 	glinet_gl-ap1300 \
 	glinet_gl-s1300 \
 	linksys_ea8300 \
 	p2w_r619ac \
-	pakedge_wr-1 \
```

Comment:
not part of this commit?

Reply:
sorry i've updated my local patch using new file, it's fixed now